### PR TITLE
Fix 174

### LINKED ISSUE #174: _userdata: use a local port between 1 and 1024

### PR DESCRIPTION
**Pull request by moul**
Fix #174

### CODE REVIEW

File: pkg/api/api.go [hunk 1]
```diff
@@ -1716,15 +1720,54 @@ func (s *ScalewayAPI) DisableAccountAPI() {
...
 // EnableMetadataAPI enable metadataAPI
 func (s *ScalewayAPI) EnableMetadataAPI() {
 	s.APIUrl = MetadataAPI
+	if os.Getenv("SCW_METADATA_URL") != "" {
+		s.APIUrl = os.Getenv("SCW_METADATA_URL")
+	}
+	s.client.Transport = &http.Transport{
+		Dial: rootNetDial,
+	}
+	s.oldTransport = &s.client.Transport
```
Comment (QuentinPerez) [inline pkg/api/api.go]: This line shouldn't before the modifications ?

Comment (moul) [inline pkg/api/api.go]: haha, yes

Comment (moul) [inline pkg/api/api.go]: fixed

**Comment by QuentinPerez** [comment]
LGTM

**Comment by aimxhaisse** [comment]
lgtm

**Pull request merged**